RecoLocalCalo/Configuration/python/hcalLocalRecoNZS_cff.py

```diff
@@ -22,11 +22,18 @@
 import RecoLocalCalo.HcalRecProducers.HBHEPhase1Reconstructor_cfi
 _phase1_hbherecoMB = RecoLocalCalo.HcalRecProducers.HBHEPhase1Reconstructor_cfi.hbheprereco.clone(
     dropZSmarkedPassed = cms.bool(False),
-    recoParamsFromDB = cms.bool(False),
     algorithm = dict(
         useM2 = cms.bool(False),
         useM3 = cms.bool(False)
     ),
+    setNegativeFlagsQIE8 = cms.bool(False),
+    setNegativeFlagsQIE11 = cms.bool(False),
+    setNoiseFlagsQIE8 = cms.bool(True),
```

Comment:
"setNoiseFlagsQIE8" switch turns on certain filters in the rechit processing. These filters, in turn, set certain rechit flags. It looks like for NZS mode only this particular filter can provide some useful information (at least this is what Olga told me), while the parameters of this filter have not been derived yet for QIE11. 

The use of filters is unrelated to choosing reco algorithm parameters from db or from cfi. This is an orthogonal issue.